build/yamls/base/conf/antrea-agent.conf

```diff
@@ -186,6 +186,10 @@ nodePortLocal:
 # 3. The Node IP
 #transportInterface:
 
+# The names of the interfaces on Node which are connected to external devices to the antrea clusters.
+# The are used to route multicast traffic when Multicast feature is enabled.
+#externalMulticastInterfaces: []
```

Comment:
If there is no internal multicast interfaces, could we just call it MulticastInterfaces?

Reply:
No sure this is a good idea because Antrea-gw0 is also a multicast interface.

Reply:
Updated. Changed it to multicastInterfaces